StudyBuddy/Hubs/ChatHub.cs

```diff
@@ -58,9 +50,6 @@ public async Task SendMessageToGroup(Guid groupName, string message)
 
         ChatMessage chatMessage = new(message, DateTime.UtcNow.AddHours(2), senderId, groupName);
```

Comment:
Account for timezones correctly

[https://learn.microsoft.com/en-us/dotnet/standard/datetime/converting-between-time-zones#converting-utc-to-local-time](https://learn.microsoft.com/en-us/dotnet/standard/datetime/converting-between-time-zones#converting-utc-to-local-time)